app/connectors/BusinessCustomerFrontendConnector.scala

```diff
@@ -17,9 +17,10 @@
 package connectors
 
 import config.ApplicationConfig
+
 import javax.inject.Inject
 import play.api.mvc.Request
-import uk.gov.hmrc.http.HttpResponse
+import uk.gov.hmrc.http.{HttpResponse, StringContextOps}
```

Comment:
This is showing as unused import when compiling code - so is it actually needed ?

Reply:
Removed import and still compiles and runs tests